[2025-04-22T00:27:05.604+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-22T00:27:05.609+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: forecast_stock_price.predict manual__2025-04-22T00:25:21.468058+00:00 [queued]>
[2025-04-22T00:27:05.612+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: forecast_stock_price.predict manual__2025-04-22T00:25:21.468058+00:00 [queued]>
[2025-04-22T00:27:05.612+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-22T00:27:05.615+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): predict> on 2025-04-22 00:25:21.468058+00:00
[2025-04-22T00:27:05.619+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'forecast_stock_price', 'predict', 'manual__2025-04-22T00:25:21.468058+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/forecast_stock_price.py', '--cfg-path', '/tmp/tmpd3kwei70']
[2025-04-22T00:27:05.621+0000] {standard_task_runner.py:105} INFO - Job 155: Subtask predict
[2025-04-22T00:27:05.620+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=460) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-22T00:27:05.621+0000] {standard_task_runner.py:72} INFO - Started process 463 to run task
[2025-04-22T00:27:05.636+0000] {task_command.py:467} INFO - Running <TaskInstance: forecast_stock_price.predict manual__2025-04-22T00:25:21.468058+00:00 [running]> on host fbe94ad2eb9c
[2025-04-22T00:27:05.660+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='forecast_stock_price' AIRFLOW_CTX_TASK_ID='predict' AIRFLOW_CTX_EXECUTION_DATE='2025-04-22T00:25:21.468058+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-22T00:25:21.468058+00:00'
[2025-04-22T00:27:05.661+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-22T00:27:05.664+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-04-22T00:27:05.664+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-22T00:27:05.665+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-22T00:27:29.073+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-22T00:27:29.941+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-22T00:27:29.943+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-22T00:27:29.958+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-22T00:27:29.959+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=forecast_stock_price, task_id=predict, run_id=manual__2025-04-22T00:25:21.468058+00:00, execution_date=20250422T002521, start_date=20250422T002705, end_date=20250422T002729
[2025-04-22T00:27:30.016+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-22T00:27:30.029+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-22T00:27:30.030+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
